src/gradualizer_cli.erl

```diff
@@ -24,6 +26,8 @@ start_application(Opts) ->
     %% If gradualizer is run as an escript this should not be necessary, but better safe than sorry.
     ok = application:load(gradualizer),
     application:set_env(gradualizer, options, Opts),
+    %% We could start the tracer based on a CLI flag, but it's config is compile-time anyway.
+    %gradualizer_tracer:start(),
```

Comment:
Is this tracing enabled by un-commenting this line?

It doesn't seem very convenient to me...

Reply:
Given the tracer has to be customised in code to make it useful for a practical test case / found bug, I think that uncommenting this single line is a minor detail :) Seriously, debugging the infinite loops, for example, took me literally hours of working with different examples, reading the code, comparing against hypothetical fixes on various branches, etc. Uncommenting this line IMO is not a blocker.